# Default initial positions for dast_1's ros2_control fake system

initial_positions:
  joint_1: 90
  joint_2: 90
  joint_3: 90
  joint_4: 90
  joint_5: 90